ui/widgets/autocomplete.js

```diff
@@ -223,6 +223,9 @@ $.widget( "ui.autocomplete", {
 			.menu( "instance" );
 
 		this._addClass( this.menu.element, "ui-autocomplete", "ui-front" );
+		if ( this.isRtl() ) {
+			this._addClass( this.menu.element, "ui-autocomplete-menu-rtl" );
```

Comment:
Well, I guess it depends on whether this will be used exclusively on generated content, which it might. If it is, then just setting the attribute is fine since we don't need to clean it up. Cleaning up the class would be easier.